sphinx/source/index.rst

```diff
@@ -1,6 +1,8 @@
 Welcome to Bokeh
 ================
 
+**A framework for interactive browser-based visualizations**
+
```

Comment:
Gotta talk to Michele Chambers about that. These are our standard Continuum
product descriptors.

_______________________________

Kerry Watson
Documentation manager

[image: Continuum Analytics] <http://continuum.io>

On Mon, Sep 12, 2016 at 3:09 PM, Bryan Van de Ven <notifications@github.com>
wrote:

> In sphinx/source/index.rst
> <https://github.com/bokeh/bokeh/pull/5150#discussion_r78444934>:
>
> > @@ -1,6 +1,8 @@
> >  Welcome to Bokeh
> >  ================
> >
> > +**A framework for interactive browser-based visualizations**
> > +
>
> I'm incredibly loathe to use the word "framework" anywhere. It's laden
> with baggage and meanings we surely do not intend. "platform" is probably a
> better choice but then the text below should also be updated accordingly.
> I'm also not sure about how this will look visually, can you attach a
> screen shot?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bokeh/bokeh/pull/5150/files/5cd10f2c6ae4a34663f9d1cda9310a21f905cbcb#r78444934>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEXgNh2FE80mvCrue44UgzgLtefbzjWks5qpbFsgaJpZM4J6-MO>
> .
>